recipes/flex/all/conandata.yml

```diff
@@ -2,3 +2,8 @@ sources:
   "2.6.4":
     sha256: "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995"
     url: "https://github.com/westes/flex/releases/download/v2.6.4/flex-2.6.4.tar.gz"
+patches:
+  "2.6.4":
+    - patch_file: patches/darwin-newer-versions.patch
+      patch_description: "https://github.com/westes/flex/issues/509"
```

Comment:
yay for mentioning the upstream issue!

Reply:
https://github.com/westes/flex/issues/509 calls out https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff but we are not aplying the same patch 😕